contracts/ExpectedRate.sol

```diff
@@ -76,6 +97,13 @@ contract ExpectedRate is Withdrawable, ExpectedRateInterface, Utils2 {
         return (expectedRate, slippageRate);
     }
 
+    function checkKncArbitrageRate(uint currentKncToEthRate) public view returns(bool) {
+        uint converseRate; uint slippage;
+	(converseRate, slippage) = getExpectedRate(ETH_TOKEN_ADDRESS,knc,10**18,true);
```

Comment:
coding style.
space after every comma
you can use: KNC_QTY_FOR_FEE_BURNER here?